remote/main.go

```diff
@@ -0,0 +1,142 @@
+// Command remote demonstrating how to connect to an existing chromium instance using chromedp.NewRemoteAllocator
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/chromedp/chromedp"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+// WebsocketTargetJson models the responses given by /json or /json/list
+// see https://chromedevtools.github.io/devtools-protocol/#endpoints for more details
+type WebsocketTargetJson struct {
+	WebSocketDebuggerUrl string `json:"webSocketDebuggerUrl"`
+	Url                  string `json:"url"`
+}
+
+// replaceResolvedHostnameIfNeeded tries to resolve the hostname in the given URL in order to work around mitigations
+// introduced because of https://bugs.chromium.org/p/chromium/issues/detail?id=813540
+//
+// Chromium will return the error "Host header is specified and is not an IP address or localhost" if the
+// DevTools are connected to via a hostname other than localhost.
+func replaceResolvedHostnameIfNeeded(url *url.URL) error {
```

Comment:
chromedp already does this; remove this function

Reply:
Function removed